python/mxnet/gluon/model_zoo/text/base.py

```diff
@@ -72,6 +72,8 @@ def get_rnn_cell(mode, num_layers, num_hidden,
                 cell = rnn.LSTMCell(num_hidden)
             elif mode == 'gru':
                 cell = rnn.GRUCell(num_hidden)
+            elif mode == 'awd-lstm':
+                cell = rnn.LSTMCell(num_hidden)
```

Comment:
New mode is not needed.

Reply:
there shouldn't be a mode specific for awd in this utility function. whether to apply weight drop or not should only depend whether the caller of this function requested non-zero weight-drop.